package.json

```
@@ -30,22 +30,21 @@
   "homepage": "https://github.com/protacon/ng-virtual-keyboard#readme",
   "prefer-stable": true,
   "dependencies": {
-    "@angular/animations": "^7.2.15",
-    "@angular/cdk": "^7.3.7",
-    "@angular/flex-layout": "^7.0.0-beta.24",
-    "@angular/material": "^7.3.7",
+    "@angular/animations": "^8.2.14",
```

Hmm, I think that we should use 
```suggestion
    "@angular/animations": "^8.2.0",
```

So that we don't force people to update to specified version - WDYT?

---

I agree, this was an oversight. I'll do some testing with this.